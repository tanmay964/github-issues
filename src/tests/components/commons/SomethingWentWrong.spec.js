import React from 'react';
import renderer from 'react-test-renderer';
import SomethingWentWrong from '../../../components/commons/SomethingWentWrong';

it('renders correctly', () => {
  const tree = renderer
    .create(<SomethingWentWrong />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
