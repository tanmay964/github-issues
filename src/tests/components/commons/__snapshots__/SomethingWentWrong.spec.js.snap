// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa sc-htpNat fgnrUs"
>
   
  <h1
    className="sc-bxivhb hFhvax"
  >
    Oops, Something went wrong!!!
  </h1>
</div>
`;
